stable/pipelines/values.yaml

```diff
@@ -0,0 +1,64 @@
+crd:
+  create: false
+
+rbac:
+  create: true
+
+deployment:
+  create: true
+
+credentials:
+  accessKey: provideMe
+
+port:
+  nginxPort: 8081
+
+storage:
+  s3:
+    bucketName: bigdata
```

Comment:
This is also a container name. so `s3.something` is not a good name
Should be:
```
something:
  containerName: ..
somethingElse:
  containerName: ....
```
Not sure what those things are (mysql / webapi ?) or something more meaningful as to how ml-pipelines is USING these containers
Maybe it should all be on one container? It depends on your understanding of how these containers are used and what's stored there.

Reply:
done